Makefile

```diff
@@ -13,3 +13,8 @@ security_check:
 lint:
 	pipenv run flake8
 .PHONY: lint
+
+init:
```

Comment:
I'd rather stick to convention and use `install:` here.

Reply:
Actually, after thinking about it, I'm onboard with `init` for this task and use `install` for installing the CLI from source.